### ISSUE TITLE
Install failed, Cannot assign type 'shared_ptr[const PCLPointCloud2]' to 'PCLPointCloud2ConstPtr' 

### ISSUE THREAD

**Issue by Tonsty** [cmpute/pcl.py#16]
cython has been upgraded to 0.29.28,
the problem still exists

**Comment by cmpute** [comment]
Which version of pcl.py are you installing? From PyPI or from the master branch?

**Comment by Tonsty** [comment]
@cmpute, I have found the evil. It is that two different versions of cython are installed on my system, of which one is the 0.26.1 at the system path '/usr/bin/cython' and the other is 0.29.28 at the local path '/home/{USER}/.local/bin/cython'. The local one is installed by 'pip3 install cython'. Since the local bin path has been added to $PATH, when I typed in command 'cython --version', it outputed '0.29.28'. However, when I install pcl-py in the system python enviroment (/usr/bin/python3),  'FindCython.cmake' will lead to '/usr/bin/cython' which is at the same directory as  '/usr/bin/python3'.

Finally, I solved the problem by installing pcl-py in an anconda enviroment.
